README.md

```diff
@@ -464,7 +446,7 @@ maintaining the Node.js project.
 
 ### Release Team
 
-Node.js releases are signed with one of the following GPG keys:
+Ayo.js releases were signed with one of the following GPG keys:
```

Comment:
I realise that this PR is currently a find and replace, but I think my above comment is worth doing now.